docs/docusaurus/src/theme/DocCategoryGeneratedIndexPage/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import DocCategoryGeneratedIndexPage from '@theme-original/DocCategoryGeneratedIndexPage';
+
+export default function DocCategoryGeneratedIndexPageWrapper(props) {
+  return (
+    <div className="generated-index-page">
+      <DocCategoryGeneratedIndexPage {...props}/>
+    </div>
```

Comment:
I have aversion towards extra divs. does it work if we move the className to `DocCategoryGeneratedIndexPage`? you'll might need to also pass any extra classNames from prop but I'm not sure. 
```suggestion
      <DocCategoryGeneratedIndexPage  className="generated-index-page" {...props}/>
```

Reply:
I thought the exact same thing, but unfortunately I don't see a clean way to do this. I've looked at the code from the component and it does not accept classNames as props, it just uses styles from its stylesheet. I could go through the way of ejecting the component in order to modify either the js of the component itself or its stylesheet, but I thought this approach was safer as we're trying to reduce swizzling of unsafe compoenents to the minimum (and this is one of them).